src/SIOReader.cc

```diff
@@ -0,0 +1,148 @@
+// podio specific includes
+#include "podio/SIOReader.h"
+
+#include "podio/CollectionIDTable.h"
+#include "podio/CollectionBase.h"
+#include "podio/EventStore.h"
+#include "podio/SIOBlock.h"
+#include "sio/definitions.h"
+
+#include <memory>
+#include <sio/compression/zlib.h>
```

Comment:
Does this mean SIO has to be build with the builtin zlib? Because similar things have caused problems in the past (https://github.com/AIDASoft/DD4hep/issues/567)

Reply:
No, SIO can be built with its builtin zlib, but it can also use the system one. For actual usage it should not matter which zlib is actually used in the end. I have briefly tested this with rebuilding SIO with the system zlib, and for podio the tests still work.

Reply:
Ok, thanks for checking.